Sources/Stripe/Models/Plans/BillingScheme.swift

```diff
@@ -35,8 +40,13 @@ extension StripePlan {
     }
     
     public struct UsageTransformation: Codable {
-        var divideBy: Int?
-        var round: RoundMode?
+        public var divideBy: Int
+        public var round: RoundMode
+        
```

Comment:
@baarde it’s probably better to leave those properties optional. There’s been so many issues lately with stripe returning null for properties, and they don’t give a definitive list of what properties returned from the API are guaranteed to be returned, and we’d rather be safe than sorry.

Reply:
`divide_by` and `round` are marked as required when creating a Plan with `transform_usage`. I'm quite confident they cannot be null. However I'll revert the change.